src/schema/python/schemaMigMgr.py

```diff
@@ -116,41 +156,57 @@ class Migration:
     """
 
     @property
-    def dirname(self):
+    def dirname(self) -> str:
         return os.path.dirname(self.filepath)
 
-    def __init__(self, from_version, to_version, name, filepath):
+    def __init__(
+        self,
+        from_version: Union[int, Type[Uninitialized]],
+        to_version: Union[int, type[Uninitialized]],
```

Comment:
type - > Type

Will we ever want to migrate _to_ Uninitialized?

Reply:
Yes, I think `to_version` wants to be an `int`, I doubt we ever want to migrate to `Uninitialized`.